pkg/name/errors.go

```diff
@@ -28,13 +27,7 @@ func (e *ErrBadName) Error() string {
 	return e.info
 }
 
-// NewErrBadName returns a ErrBadName which returns the given formatted string from Error().
-func NewErrBadName(fmtStr string, args ...interface{}) *ErrBadName {
+// newErrBadName returns a ErrBadName which returns the given formatted string from Error().
+func newErrBadName(fmtStr string, args ...interface{}) *ErrBadName {
```

Comment:
This is technically breaking but do we care?

Reply:
I think this was a mistake to export, and can't see a situation where an external user would want to create new `ErrBadName`s.

I feel somewhat less sure about removing `IsErrBadName`, since someone might have used that.